src/main/charts/confluence/Changelog.md

```diff
@@ -112,19 +112,19 @@ index a39d4ae..8e148f3 100644
 
 
 * Version 0.9.0 
-* Merge branch 'master' into DCKUBE-317-additional-libraries-for-synchron 
```

Comment:
Just wondering if we should be updating these changelogs? I can understand changelogs using `main` once this PR has been merged, but for historical reasons and provenance I think we should keep the existing changelogs as is.

Reply:
Yes, however the master branch is not existed anymore and this is renamed to main - do you think we should still keep master?